src/procedure/nodes/registry.cpp

```diff
@@ -119,6 +120,8 @@ ProcedureNodeRegistry::ProcedureNodeRegistry()
     registerProducer<SimpleRestraintPotentialProcedureNode>(ProcedureNode::NodeType::SimpleRestraintPotential,
                                                             "Restraint atoms of molecules to their current positions",
                                                             "Potentials");
+    // Sites
```

Comment:
Man, I am _fussy_ when it comes to blank lines.... :smiley:
```suggestion

    // Sites
```